packages/components/src/settings-form/index.tsx

```diff
@@ -31,7 +47,7 @@ export const SettingsForm = ( {
 	onChange = () => {},
 	validate = () => ( {} ),
 	buttonLabel = __( 'Proceed', 'woocommerce-admin' ),
-} ) => {
+}: SettingsFormProps ): JSX.Element => {
```

Comment:
So we haven't done a lot of this, but a more succinct type definition for a function component can use the `React.FC` type, which handles things like the return type and also supports props like `children`.

Here's an example:

```suggestion
export const SettingsForm: React.FC<SettingsFormProps> = ({...}) =>
```